lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextLifeCycle.cs

```diff
@@ -71,10 +71,10 @@ public void Start()
             IContextStart contextStart = new ContextStartImpl(Id);
             
             // TODO: enable
-            //foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
-            //{
-            //    startHandler.OnNext(contextStart);
-            //}
+            //// foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
```

Comment:
Then, finally,
* // : non-documentation comments 
* /// : documentations (xml)
* //// : no leading space (except code indentation)

I will rearrange all the code again with above rules.